modules/portal/portal-impl/src/main/java/com/enonic/xp/portal/impl/handler/mapping/MappingFilterHandlerWorker.java

```diff
@@ -41,7 +41,10 @@ public PortalResponse execute()
         final Trace trace = Tracer.current();
         if ( trace != null )
         {
-            trace.put( "contentPath", this.request.getContent().getPath().toString() );
+            if ( this.request.getContent() != null )
+            {
+                trace.put( "contentPath", this.request.getContent().getPath().toString() );
```

Comment:
request has `contentPath()` method that makes null check unneeded.

Reply:
Thank you! Did it.